docs/operations/api-reference.md

```diff
@@ -927,6 +935,11 @@ Returns segment information lists including server locations for the given query
 
 #### GET
 
+> Note: Much of this information is available in a simpler, easier-to-use form through the Druid SQL
+> [`INFORMATION_SCHEMA.TABLES`](../querying/sql.md#tables-table),
+> [`INFORMATION_SCHEMA.COLUMNS`](../querying/sql.md#columns-table), and
+> [`sys.segments`](../querying/sql.md#segments-table) tables.
```

Comment:
nit: Is there a reason why one would choose these APIs over what's available via Druid SQL or vice versa.

Reply:
The reason lies in the words "Much of this information". Some info is only available in the JSON APIs. (I didn't go through and enumerate which specific info that is, though.)